userguide/virtualmachines.rst

```diff
@@ -81,6 +81,8 @@ are available:
 
 * :guilabel:`Stop` shuts down the VM.
 
+* :guilabel:`Restart` shuts down and immediately starts the VM.
```

Comment:
I added a description for this option as well in PR #765. Be aware that other PR's prob conflict.

Reply:
Let's keep this PR open and just back that change out of #765 . That will prevent having to update all the Redmine tickets and doc queue.